lib/pf/cmd/pf/fixpermissions.pm

```diff
@@ -0,0 +1,74 @@
+package pf::cmd::pf::fixpermissions;
+=head1 NAME
+
+pf::cmd::pf::fixpermissions add documentation
+
+=head1 SYNOPSIS
+
+ pfcmd fixpermissions
+
+Fix the permissions of the files and directories of packetfence
+
+=head1 DESCRIPTION
+
+pf::cmd::pf::fixpermissions
+
+=cut
+
+use strict;
+use warnings;
+use base qw(pf::cmd);
+use pf::file_paths;
+
+sub parseArgs { 1 }
+
+sub _run {
+    my $pfcmd = "${bin_dir}/pfcmd";
+    _changeFilesToOwner('pf',@log_files, @stored_config_files, $install_dir, $bin_dir, $conf_dir, $var_dir, $lib_dir, $log_dir, $generated_conf_dir, $tt_compile_cache_dir);
```

Comment:
Would it not be better to make the owner a configuration item?
Or do we end up in a chicken and egg case?